charts/newrelic-logging/templates/daemonset.yaml

```
@@ -134,9 +162,18 @@ spec:
         - name: fluent-bit-config
           configMap:
             name: {{ template "newrelic-logging.fluentBitConfig" . }}
-        - name: var
+        - name: logs
           hostPath:
-            path: /var
+            path: {{ .Values.fluentBit.linuxMountPath | default "/var" }}
```

```suggestion
            path: /var/log
```

And then a separate mount for `/var/lib/docker/containers`?

---

In that case we need to skip mounting it on GKE autopilot anyways because /var/lib is not allowed to be mounted. So instead linuxMountPath we should have skipMountingVarLib or something that have not much sense to me